care/facility/api/serializers/patient.py

```diff
@@ -539,6 +546,11 @@ def create(self, validated_data):
             # If the user is not a doctor then the user type is the same as the user type
             validated_data["user_type"] = user_type
 
+        if validated_data.get("reply_to"):
+            validated_data["reply_to"] = PatientNotes.objects.get(
+                external_id=validated_data["reply_to"]
+            )
+
```

Comment:
Add conditions in `get` to check if the `reply_to` belongs to the same thread and consultation

Reply:
added